docs/developer_setup.md

```diff
@@ -0,0 +1,55 @@
+# Developer Setup
+
+Logging in to the SUI requires a running instance of ManageIQ. Instructions on how to install ManageIQ can be found
+[here](https://github.com/ManageIQ/guides/blob/master/developer_setup.md).
+
+### Install Repository and Prerequisites
+
+- Have the [ManageIQ](http://github.com/ManageIQ/manageiq) repo cloned into a
+  directory named `manageiq`, and ready to be started. See [here](https://github.com/ManageIQ/guides/blob/master/developer_setup.md)
+  for the steps required to setup ManageIQ.
+- Have this repo as a sibling to the `manageiq` directory:
+`git clone git@github.com:ManageIQ/manageiq-ui-service.git`.
+- Have nodejs **v6.7.0** and npm **3.10.3** installed (npm should be installed with NodeJS)
```

Comment:
@chriskacerguis oh yeah now i remember, if ya don't have  npm installed ya can't install yarn and gulp (next step) so still kinda required for build